glib/src/auto/source.rs

```diff
@@ -45,6 +45,13 @@ impl Source {
         }
     }
 
+    #[cfg(feature = "v2_86")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_86")))]
+    #[doc(alias = "g_source_dup_context")]
+    pub fn dup_context(&self) -> Option<MainContext> {
```

Comment:
Should implement the normal context getter in terms of this if building for new enough GLib. The normal getter is unsound.